Doc/library/importlib.resources.rst

```diff
@@ -182,4 +215,8 @@ The following functions are available.
     *package* is either a name or a module object which conforms to the
     ``Package`` requirements.
 
-   .. deprecated:: 3.11
+    .. deprecated:: 3.11
+
+       Calls to this function can be replaced by::
+
+          (resource for resource in files(package).iterdir() if resource.is_file())
```

Comment:
Not entirely sure on this one, but the `contents()` function was documented to return an `Iterator[str]` -- would `resource.name` or `str(resource)` be best?
```suggestion
          (resource.name for resource in files(package).iterdir() if resource.is_file())
```